src/fileio/datafile.cxx

```diff
@@ -61,12 +62,17 @@ Datafile::Datafile(Options *opt) : parallel(false), flush(true), guards(true), f
   OPTION(opt, enabled, true);
   OPTION(opt, init_missing, false); // Initialise missing variables?
   OPTION(opt, shiftOutput, false); //Do we want to write 3D fields in shifted space?
+
+  writeFlushCounter = 0;
+  OPTION(opt, writeFlushFreq, 1); //How frequently do we flush the file
```

Comment:
This was to match the style of the other options in this section, where the default is given again (e.g. `enabled`), but happy to change if preferred.